src/layout/Footer/Footer.svelte

```diff
@@ -117,6 +117,12 @@
 			</Button>
 		{/each}
 	</div>
+	<div class="column">
+		<p>Support</p>
+		<Button variant="hyperlink" href="/privacy">
+			Privacy policy
```

Comment:
```suggestion
			Privacy
```
Can you also localize the text?